epochStart/bootstrap/process.go

```diff
@@ -466,6 +466,11 @@ func (e *epochStartBootstrap) requestAndProcessing() (Parameters, error) {
 		}
 	}
 
+	err = e.messenger.CreateTopic(core.ConsensusTopic+e.shardCoordinator.CommunicationIdentifier(e.shardCoordinator.SelfId()), true)
```

Comment:
Although this solves the issue, we might think of a solution in which, after the creation of the shard coordinator instance, the node automatically registers to the consensus topic. In this way we won't "forget" to create the consensus topic on other possible cases.